### PR TITLE
More edits to readme

### PR DESCRIPTION
**Pull request by duglin**
Move stuff we're not doing into the docs/future dir

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,146 +2,40 @@
...
-   Service Broker that can then be registered into a Service Catalog - 
-   either a Kubernetes Service Catalog or a Cloud Foundry one.
-2. Provide a mechanism by which a Kubernetes managed Service Broker can be
-   told which Namespace into which a new service instance should be created.
-   This is optional because not all services necessarily deploy new running
-   code for each service instance. For example, a DB service may simply
-   create a new set of DB tables per instance.
-
-#### Service Recipes
+The exact purpose of thie SIG will grow over time, and as such, please see
```
Comment (pmorie) [inline README.md]: typo: the

File: README.md [hunk 2]
```diff
@@ -2,146 +2,40 @@
...
+the various "version" folders under the [docs](./docs) directory for the
+exact list of use-cases, features and design decisions that have been made.
```
Comment (pmorie) [inline README.md]: made -> agreed to

File: README.md [hunk 3]
```diff
@@ -2,146 +2,40 @@
...
-   resources that can then be instantiated into a Kubernetes environment.
-   Note that these may or may not be "Services" as defined above.
-2. These deployments, if they support the Service Broker APIs, can then be
-   registered into the Service Broker for use by applications on the
-   platform.
-
-The SIG will also monitor and discuss the progress made by the
-[Service Broker working group](https://github.com/servicebroker/servicebroker).
+However, this SIG is (initially) focused on an implementation of the
+CloudFoundry Service Broker APIs. This would include the ability for Service
```
Comment (pmorie) [inline README.md]: Cloud Foundry

File: README.md (hunk 1 above)
Comment (duglin) [inline README.md]: fixed

File: README.md (hunk 2 above)
Comment (duglin) [inline README.md]: done

File: README.md (hunk 3 above)
Comment (duglin) [inline README.md]: done, and got another one too

**Comment by duglin** [comment]
updated, PTAL

**Comment by pmorie** [comment]
LGTM

**Pull request merged**